package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "setup-packer",
-  "version": "2.1.0",
+  "version": "3.0.0",
```

Comment:
https://github.com/hashicorp/setup-packer/pull/90#issue-2107159114 see @ahanoff's comment referencing GitHub's suggestion to make this change a major version bump in actions.